src/libcrun/container.c

```diff
@@ -1074,6 +1074,15 @@ container_init_setup (void *args, pid_t own_pid, char *notify_socket, int sync_s
   cleanup_free char *rootfs = NULL;
   int no_new_privs;
 
+#if HAVE_DLOPEN && HAVE_LIBKRUN
+  if (entrypoint_args->context->is_krun == true)
+    {
+      ret = libcrun_configure_libkrun (args, err);
+      if (UNLIKELY (ret < 0))
+        return ret;
+    }
+#endif
+
   ret = libcrun_configure_handler (args, err);
```

Comment:
we need to make sure that when a global handler is set, the `libcrun_configure_handler` fails if there is another handler specified in the container spec (or accept when it is the same handler specified).

Reply:
@giuseppe This is resolved in latest commit. Just confused about the **err code**  i am thinking of `EACCES, "invalid attempt to override already configured global handler: %s"`

Reply:
@giuseppe does `EACCES` looks fine from your side ?